### DISCUSSION TITLE
tsserver.js path

### DISCUSSION THREAD

**Discussion by m-anish-gupta** [typescript-language-server/typescript-language-server#856] [Q&A]
In the initialize options, it is specified to give the path to the `tsserver.js` file but where is that file located? I installed the language server using `npm install -g typescript-language-server typescript`

**Comment by rchl** [comment]
You shouldn't need to manually set it. It should be auto-discovered either from your global `typescript` package or from you project.

But if you must set it then it's typically in `typescript/lib/tsserver.js`.

**Reply by m-anish-gupta** [reply]
Yes I found it inside the node modules directory in cwd. Thanks!

**Comment by m-anish-gupta** [comment]
Yes I found it inside the node modules directory in cwd. Thanks!
